function-images/tests/chained-function-eventing/producer/main.go

```diff
@@ -49,25 +48,27 @@ type server struct {
 	UnimplementedGreeterServer
 }
 
-const workflowId = "producer.chained-functions-eventing.192.168.1.240.sslip.io"
+type vHiveMetadata struct {
+	WorkflowId string `json:"WorkflowId"`
+	InvocationId string `json:"InvocationId"`
+	InvokedOn time.Time `json:"InvokedOn"`
+}
```

Comment:
this should be inside the metadata module. the function's user code should not know about it.